config/default.yml

```diff
@@ -770,6 +770,8 @@ Style/SpaceAroundOperators:
 
 Style/SpaceBeforeBlockBraces:
   EnforcedStyle: space
+  # When true, ->{ ... } is allowed.
+  ExceptAfterStabbyLambda: false
```

Comment:
This definitely needs a more clear name. Stabby lambda is a term for the entire lambda literal. And I'd prefer something which starts with "Allow" instead of "Except".